### ISSUE TITLE
fastlane 1.84.0 breaks badge action

### ISSUE THREAD

**Issue by denbec** [fastlane/fastlane#4501]
### New Issue Checklist

- [x] Updated fastlane to the latest version

### Issue Description

There is a conflict in mini_magick. I don't have enough knowledge of ruby and gems to know if I could fix this by myself.

##### Complete output when running fastlane, including the stack trace and command used

```
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:2007:in `raise_if_conflicts': Unable to activate badge-0.3.0, because mini_magick-4.5.1 conflicts with mini_magick (~> 4.0.2) (Gem::LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1176:in `activate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems.rb:186:in `rescue in try_activate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems.rb:183:in `try_activate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:132:in `rescue in require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:144:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/actions/badge.rb:6:in `run'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/runner.rb:143:in `block (2 levels) in execute_action'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/actions/actions_helper.rb:35:in `execute_action'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/runner.rb:129:in `block in execute_action'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/runner.rb:128:in `chdir'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/runner.rb:128:in `execute_action'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/fast_file.rb:149:in `method_missing'
	from Fastfile:80:in `block (2 levels) in parsing_binding'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/lane.rb:36:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/lane.rb:36:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/runner.rb:52:in `block in execute'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/runner.rb:46:in `chdir'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/runner.rb:46:in `execute'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/lane_manager.rb:46:in `cruise_lane'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/commands_generator.rb:41:in `block (2 levels) in run'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane_core-0.43.1/lib/fastlane_core/ui/fastlane_runner.rb:26:in `run!'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/commands_generator.rb:171:in `run'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/lib/fastlane/commands_generator.rb:12:in `start'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.84.0/bin/fastlane:6:in `<top (required)>'
	from /usr/local/bin/fastlane:23:in `load'
	from /usr/local/bin/fastlane:23:in `<main>'
```

### Configuration Files

Fastfile:
```ruby
  lane :beta do
    
	ensure_git_status_clean

	build_number = number_of_commits
	increment_build_number(build_number: build_number)


    match(type: "appstore", readonly: true)
	
    version = get_version_number(xcodeproj: "thanks.xcodeproj")
    build_number = number_of_commits
    badge(shield: version.to_s+"-"+build_number.to_s+"-orange", no_badge: true)

    gym(
    	scheme: 'thanksinhouse',
   		use_legacy_build_api: true,
   		include_symbols: true
    )

    pilot

    reset_git_repo
  
  end
```

### Environment

fastlane version (run `fastlane -v`): 1.84.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? No

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? No

**Comment by i2amsam** [comment]
Hi @denbec thanks for the report!  This is a conflict between frameit's `mini_magick` dependency and `badge`'s.  Badge merged a compatibility PR this morning  https://github.com/HazAT/badge/pull/22 it should be fixed in the next release. 

If you're not using `frameit` you should be able to downgrade:

```
gem uninstall mini_magick -v 4.5.1 
gem install mini_magick -v 4.0.2
```

In order to get unblocked.

**Comment by i2amsam** [comment]
`badge 0.4.1` is released now, you should be able to get it with `gem install badge`

**Comment by denbec** [comment]
Perfect, is working again. Thank you guys.
